dex/asset.go

```diff
@@ -15,10 +15,52 @@ func (err Error) Error() string { return string(err) }
 
 const (
 	UnsupportedScriptError = Error("unsupported script type")
-	LockTimeTaker          = 24 * time.Hour
-	LockTimeMaker          = 48 * time.Hour
+	lockTimeTaker          = 24 * time.Hour
+	lockTimeMaker          = 48 * time.Hour
 )
 
+var (
+	// These variables are defined to enable setting custom locktime values that
+	// will be used in place of the `lockTimeTaker` and `lockTimeMaker` constants
+	// defined above, _when_ running on a test network (testnet or regtest).
+	// Values for both variables may be set at build time using linker flags e.g:
+	// go build -ldflags "-X 'decred.org/dcrdex/dex.testLockTimeTaker=10m' \
+	// -X 'decred.org/dcrdex/dex.testLockTimeMaker=20m'"
+	// Same values should be set when building server and client binaries.
+	testLockTimeTaker string
+	testLockTimeMaker string
+)
+
+// LockTimeTaker returns the taker locktime value that should be used by both
+// client and server for the specified network. Mainnet uses a constant value
+// while test networks support setting a custom value during build.
+func LockTimeTaker(network Network) time.Duration {
+	if network == Mainnet || testLockTimeTaker == "" {
+		return lockTimeTaker
+	}
+	testLockTime, _ := time.ParseDuration(testLockTimeTaker)
```

Comment:
The user definitely wants to know if the time they specified could not be parsed.

Reply:
If `testLockTimeTaker` or `testLockTimeMaker` fail to parse (when not just `""`) it should panic.  I can think of no good reason to silently go with the default if the build command provided a string that was not a valid duration.